project2/README.md

```diff
@@ -1,16 +1,16 @@
 # Scraping the web with Python
 
-This section covers scraping pages on the Web with Python using a [get](http://www.w3schools.com/tags/ref_httpmethods.asp) request. 
+This section covers scraping pages on the Web with Python using a [get](http://www.w3schools.com/tags/ref_httpmethods.asp) request.
 
-To begin with, we will import the [modules](http://docs.python.org/3/tutorial/modules.html) or packages we need to scrape websites. This is done with a series of import statements at the top of the file. 
+To begin with, we will import the [modules](http://docs.python.org/3/tutorial/modules.html) or packages we need to scrape websites. This is done with a series of import statements at the top of the file.
 
 ```Python
 import requests
 from bs4 import BeautifulSoup
-import csv
+import CSV
```

Comment:
Are you sure the csv module is in upper case?

Reply:
Agreed. import CSV will throw a ModuleNotFoundError